### PR TITLE
Adding JLBP-13: Remove references to deprecated features quickly

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

File: library-best-practices/JLBP-13.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[JLBP-13] Remove references to deprecated features quickly
+----------------------------------------------------------
+
+- This best practice refers to code that *uses* deprecated functionality, not to
```
[the remark this answers is not in the document]
Comment (garrettjonesgoogle) [reply library-best-practices/JLBP-13.md]: It applies to a library's usage of deprecated features in its dependencies. This is relevant to every layer except leaf nodes that have no dependencies of their own. I will try to clarify this.

**Comment by garrettjonesgoogle** [comment]
Addressed feedback, PTAL

File: library-best-practices/JLBP-13.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[JLBP-13] Quickly remove references to deprecated features in dependencies
+--------------------------------------------------------------------------
+
+- This best practice refers to code that *uses* deprecated functionality from
+  another library, not to the deprecated functionality itself.
+  - Consequently, this best practice does not apply to libraries that have no
```
Comment (elharo) [inline library-best-practices/JLBP-13.md]: All libraries depend on the JDK and it does have deprecated features. 20 years in some of those are finally being removed. I suggest removing the first clause.

File: library-best-practices/JLBP-13.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    dependencies, and does not apply to usage of deprecated features within
+    a library.
+  - A "usage" in this context is any reference to a class or method in
```
Comment (elharo) [inline library-best-practices/JLBP-13.md]: I'm not sure we need to define this here. It seems self-evident in context.

File: library-best-practices/JLBP-13.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- The earlier you remove usages of external deprecated functionality, the
+  more versions of your product will work with your dependency once the
+  deprecated features are fully deleted or hidden in the dependency.
+  - For example, api-common-java 1.6.0 refers to `Futures.transform(ListenableFuture, Function)`
+    in Guava, which was deprecated in 19.0 and removed in 26.0. In
+    api-common-java 1.7, the code was changed to instead call
+    `Futures.transform(ListenableFuture, Function, Executor)`,
+    which is not deprecated. The oldest version of api-common-java that
+    can be used with Guava 26.0 or later is 1.7.0, because 1.6.0 and earlier
+    refers to a deleted method. If api-common-java had moved to the new
```
Comment (elharo) [inline library-best-practices/JLBP-13.md]: refers --> refer

File: library-best-practices/JLBP-13.md (hunk 2 above)
Comment (garrettjonesgoogle) [reply library-best-practices/JLBP-13.md]: Oh of course, I overlooked that. I'll change it.

File: library-best-practices/JLBP-13.md (hunk 3 above)
Comment (garrettjonesgoogle) [reply library-best-practices/JLBP-13.md]: Ok - I may have overcompensated based on your prior comment. I'll remove it.

File: library-best-practices/JLBP-13.md (hunk 4 above)
Comment (garrettjonesgoogle) [reply library-best-practices/JLBP-13.md]: Will fix

**Comment by garrettjonesgoogle** [comment]
Feedback addressed, PTAL

**Pull request merged**